packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -133,21 +150,29 @@ export const ComboBoxNext = forwardRef(function ComboBoxNext<T>(
     css: comboBoxNextCss,
     window: targetWindow,
   });
-  const listId = useId(ListProps?.id);
-  const listRef = useRef<HTMLUListElement>(null);
+  const {
+    id: listPropsId,
+    onChange: onListChange,
```

Comment:
`onListChange` is ignored? should `onChange`/`onSelect` of Combo Box be passed to List and used to serve the same purpose?